src/components/DpLabel/DpLabel.vue

```diff
@@ -23,7 +23,7 @@
 
 <script setup lang="ts">
 import { computed } from 'vue'
-import { CleanHtml as vCleanhtml } from '~/directives' //NOSONAR see https://sonarsource.atlassian.net/browse/JS-97
```

Comment:
Why did you remove that comment?

Reply:
because they fixed the issue and the linter should not complain anymore 
https://sonarsource.atlassian.net/browse/JS-97